test/stepDefinitions/create-caseSteps.js

```diff
@@ -14,6 +15,7 @@ var { defineSupportCode } = require("cucumber");
 defineSupportCode(function ({ Given, When, Then}) {
 
   let caseWizardPage = new CreateCaseWizardPage();
+  let createCasePage1 = new ConditionalsCreateCasePage1();
```

Comment:
We may want to rename the var to conditionalsCreateCasePage1 to reduce confusion later on when more page object classes are being called